### PR TITLE
Schedule pages

### PR DESCRIPTION
**Pull request by AntonPuko**
Schedules works for locations only atm, and it `might be` some mess with nav transitions..needs to test carefully.

i tweaked location page to be able to use tab-nav-bar:
location

schedule page:
schedule

schedules page: 
schedules

schedule form:
form

### CODE REVIEW

File: www/js/controllers/new-location.ctrl.js [hunk 1]
```diff
@@ -40,9 +40,12 @@ angular.module('brewskey.controllers').controller('NewLocationCtrl', [
       promise.then(
         function(response) {
           $scope.editing = true;
-
           $ionicHistory.currentView($ionicHistory.backView());
-          $state.go('app.location', { locationId: response.id });
+          $state.go(
+            'app.location',
+            { locationId: response.id },
+            { location: 'replace' }
```
Comment (jlkalberer) [inline www/js/controllers/new-location.ctrl.js]: replace doesn't actually do anything :/

File: www/templates/schedule.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<ion-view view-title="Schedule">  
+  <ion-content>
+    <div class="padding text-center" ng-show="loading">
+      <ion-spinner></ion-spinner>
+    </div>
+    <div class="list card" ng-show="!loading">
+    	<div class="item">
+			<h1>
```
Comment (AntonPuko) [inline www/templates/schedule.html]: ops, bad indentation

File: www/templates/schedules.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                </span>
+            </a>
+            <div class="item item-divider"></div>
+            <div class="item text-center" ng-show="loading">
+                <ion-spinner></ion-spinner>
+            </div>
+            <div ng-show="!loading">
+                <a 
+                  class="item"
+                  href="#/app/schedules/{{schedule.id}}"
```
Comment (jlkalberer) [inline www/templates/schedules.html]: nit: use `ui-sref`

File: www/templates/schedules.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                  ng-repeat="schedule in schedules" 
+                >
+                  <div><strong>Name: </strong>{{ schedule.name }}</div>
+                  <div style="white-space: normal;">
+                    <strong>Days: </strong>
+                    {{ schedule.days }}
+                  </div>
+                  <div>
+                    <strong>From: </strong>{{ schedule.startTime }}
+                    <strong>To: </strong>{{ schedule.endTime }}
```
Comment (jlkalberer) [inline www/templates/schedules.html]: This formatting is fine for now but I think we'll change it a little bit

Comment (AntonPuko) [inline www/templates/schedules.html]: Yeah, i just done something basic :) it was very lazy to think about design, when I need to learn the whole angular :)))

**Pull request merged**